### PR TITLE
Fixes #22092 - refactor lodash usage - prefer to use esw syntax

### PR DESCRIPTION
**Pull request by sharvit**
* convert imports to be more spesific
* prefer not to use lodash whenever possible

### CODE REVIEW

**Review by ohadlevy — APPROVED** [review]
changes look good, pending tests + commit message / issue. thanks!

**Review by boaz0 — APPROVED** [review]
Good job LGTM!

File: webpack/test_setup.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
+import 'babel-polyfill';
```
Comment (sharvit) [inline webpack/test_setup.js:1]: Need to add `babel-polyfill` so tests will run on `nodejs < 8`

`Object.values` and `Object.entries` are introduced in `nodejs@8`

**Comment by sharvit** [comment]
@ohadlevy - ready to merge, all tests are passing...

**Review by ohadlevy — APPROVED** [review]

**Comment by ohadlevy** [comment]
katello tests are not related - thanks @sharvit

**Pull request merged**